src/daemon/command_parser_executor.cpp

```diff
@@ -452,5 +452,27 @@ bool t_command_parser_executor::output_histogram(const std::vector<std::string>&
   return m_executor.output_histogram(min_count, max_count);
 }
 
+bool t_command_parser_executor::print_coinbase_tx_sum(const std::vector<std::string>& args)
+{
+  if(!args.size())
+  {
+    std::cout << "need block height parameter" << std::endl;
+    return false;
+  }
```

Comment:
Maybe should default to use current blockchain height, though that's a question of taste I guess